register-studies.csl

```diff
@@ -0,0 +1,1512 @@
+                      <text term="issue" form="short" text-case="capitalize-first"/>
+                    </if>
+                  </choose>
+                  <text variable="number"/>
+                </group>
+              </else>
+            </choose>
+          </group>
+          <group prefix="(" suffix=")" delimiter=" ">
+            <text variable="authority"/>
+            <choose>
+              <if variable="container-title" match="any">
+                <!--Only print year for cases published in reporters-->
+                <date variable="issued" form="numeric" date-parts="year"/>
+              </if>
+              <else>
+                <date variable="issued" form="text"/>
+              </else>
+            </choose>
+          </group>
+        </group>
+      </if>
+      <else-if type="bill legislation" match="any">
+        <group prefix=", " delimiter=" ">
+          <group delimiter=", ">
+            <choose>
+              <if variable="number">
+                <!--There's a public law number-->
+                <text variable="number" prefix="Pub. L. No. "/>
+                <group delimiter=" ">
+                  <!--change to label variable="section" as that becomes available -->
+                  <text term="section" form="symbol"/>
+                  <text variable="section"/>
+                </group>
+                <group delimiter=" ">
+                  <text variable="volume"/>
+                  <text variable="container-title"/>
+                  <text variable="page-first"/>
+                </group>
+              </if>
+              <else>
+                <group delimiter=" ">
+                  <text variable="volume"/>
+                  <text variable="container-title"/>
+                  <!--change to label variable="section" as that becomes available -->
+                  <text term="section" form="symbol"/>
+                  <text variable="section"/>
+                </group>
+              </else>
+            </choose>
+          </group>
+          <date variable="issued" prefix="(" suffix=")">
+            <date-part name="year"/>
+          </date>
+        </group>
+      </else-if>
+      <else-if type="treaty">
+        <group delimiter=" ">
+          <number variable="volume"/>
+          <text variable="container-title"/>
+          <text variable="page"/>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <!-- As per the RS stylesheet, page numbers are not proceeded by p. or pp. in in-text citations, and names and dates are only separated by a space. -->
+  <macro name="point-locators">
+    <group>
+      <choose>
+        <if locator="page" match="none">
+          <label variable="locator" form="short" suffix=" "/>
+        </if>
+      </choose>
+      <text variable="locator"/>
+    </group>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="2" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
```

Comment:
The `et-al-use-first="2"` here seems wrong to me. The RS styleguide has
> (Johnson et al. 2014: 43)

Your version produces

>(Johnson, Meyer, et al. 2014: 43)

Reply:
Thanks for picking this up. You're absolutely right, it should be: et-al-use-first="1".

Reply:
I removed the comma in front of "et al." as well.

Reply:
Thank you!